slave/slave.go

```diff
@@ -4,13 +4,13 @@ package slave
 
 import (
 	"fmt"
+	"github.com/genomagic/slave/components/quality_controller"
```

Comment:
Misplaced import, should go together with the `"github.com/genomagic..."` local block.